.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+# Empty
+version: 2
+
+jobs:
+  build:
+    parallelism: 3
+    working_directory: ~/abortioneering
+    docker:
+      - image: circleci/ruby:2.4.3-node-browsers
```

Comment:
does this conform to the yaml spec? aren't arrays supposed to be aligned with their parent key? e.g.:

```
docker:
- image: circleci/ruby:2.4.3
  environment:
    WHATEV: true
- image: circleci/mongo:latest